modules/install-central-operator.adoc

```diff
@@ -28,6 +28,20 @@ When you install {product-title} for the first time, you must first install the
 ====
 . Enter the new project name (for example, `stackrox`), and click *Create*. Red Hat recommends that you use `stackrox` as the project name.
 . Under the *Provided APIs* section, select *Central*. Click *Create Central*.
+. Optional: If you are using declarative configuration, next to *Configure via:*, click *YAML view* and add the information for the declarative configuration, such as shown in the following example:
```

Comment:
Same here regarding the notion of mentioning that they should use the names of config maps / secrets they want to use?

Reply:
added info for this